README.md

```diff
@@ -0,0 +1,188 @@
+  <ul>
+    <li><a href="#">Github</a></li>
+      <li><a href="#">Github Pages</a></li>
+  </ul>
+</details>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+>You need to have Code Editor Installed.
+>Make sure Nodejs and NPM is installed in your system for linters.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+git clone https://github.com/ghreza-crypto/math-magicians.git
+```
+2. Open terminal and cd into project
+```bash
+cd math-magicians
+```
+3. Run the project with:
+```bash
+npm start
+```
+
+### Install
+
+Install this project with: linters
+
+To check linters run the following command:
+1. Install WebHint:
+```bash
+npm install --save-dev hint@7.x
+```
+2. Install StyleLint
+```bash
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x 
+```
+3. Install eslint
+```bash
+npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x 
+```
+## Check Linters Error
+1. To check CSS or SCSS linters error:
+```bash
+npx stylelint "**/*.{css,scss}"
+```
+2. To fix CSS or SCSS linters error:
+```bash
+npx stylelint "**/*.{css,scss}" --fix
+```
+3. To check HTML linters error
+```bash
+npx hint .
+```
+4. To check JavaScript linters error
+```bash
+npx eslint .
+```
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[using its calculator]**
+- **[single page application which makes using easy]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[mobile version functionality]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+>
+
+👤 **Ghulam Reza Rajabi**
+
+- GitHub: [@githubhandle](https://github.com/ghreza-crypto)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing
+
+ Contributions, issues, and feature requests are welcome!
+- Feel free to check the [issues page](https://github.com/ghreza-crypto/to-do-list/issues)
+
+## ⭐️ Show your support <a name="support"></a>
+
+> 
+
+If you like this project give it a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank from Microverse and all of my teammates.
+
+- [Microverse Videos](https://www.youtube.com/watch?v=PrIY8sYwe90&ab_channel=Microversevideos)
+- [Setup Linters](https://questions.microverse.org/t/configure-linters-for-html-and-css/2009)
+- [Mobile First](https://www.mightyminnow.com/2013/11/what-is-mobile-first-css-and-why-does-it-rock/)
+- [Figma Templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
```

Comment:
Great job @ghreza-crypto 🤝

- [ ] [Optional] Kindly ensure these acknowledgments are relevant to this project. You may remove the `Mobile First`, and `Figma` links from this section.